src/screens/OtherProfile/index.js

```diff
@@ -479,13 +404,13 @@ const OtherProfile = () => {
     try {
       const processPostApi = await unblockUserApi({userId: dataMain.user_id});
       if (processPostApi.code === 200) {
-        checkUserBlockHandle(dataMain.user_id);
+        refetchBlockStatus();
         blockUserRef.current.close();
         specificIssueRef.current.close();
         reportUserRef.current.close();
       }
     } catch (e) {
-      checkUserBlockHandle(dataMain.user_id);
+      refetchBlockStatus();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `checkUserBlockHandle` has been replaced with `refetchBlockStatus`. Ensure that `refetchBlockStatus` is providing the same functionality as `checkUserBlockHandle`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->